config/deploy.yml

```diff
@@ -0,0 +1,30 @@
+service: ruby-europe
+image: rubyeurope/rubyeurope.com
+
+servers:
+  web:
+    hosts:
+      - 64.227.117.90
+proxy:
+  app_port: 3000
+  ssl: true
+  host: do.rubyeurope.com
```

Comment:
since we're merging to `main`, it should probably be just rubyeurope.com, right?